### DISCUSSION TITLE
can I use gradient in shadow color like I can in border color?

### DISCUSSION THREAD

**Discussion by XxRiKxX** [hyprwm/Hyprland#3232] [General]
title

**Comment by ArtemSmaznov** [comment]
linking related issue https://github.com/hyprwm/Hyprland/issues/3816

**Comment by vaxerski** [comment]
should be simple actually. https://github.com/hyprwm/Hyprland/issues/10492

**Comment by Piotr-Adamm** [comment]
<img width="1672" height="1591" alt="image" src="https://github.com/user-attachments/assets/72008962-96dc-4d7b-82d4-54007eefae10" />


Has this feature been removed?

**Reply by chocoblocko9** [reply]
no you're probably just not on -git

**Reply by Piotr-Adamm** [reply]
yeah you're right
